test/pages/box_buster.spec.js

```diff
@@ -64,227 +43,276 @@ describe('BoxBuster', () => {
-    const wrapper = mount(BoxBuster, { localVue })
-    await wrapper.setData({ labwhereResponse: { success: null } })
-    expect(wrapper.find('table').exists()).toBe(false)
+  it('is empty on start', () => {
+    expect(wrapper.find('tbody').findAll('tr')).toHaveLength(1)
+    expect(wrapper.find('tbody').text()).toContain(MSG_NO_RECORDS)
   })
 
-  it('sorts list of plates by plateMap and positive count', async () => {
-    const data = { plates: examplePlates }
-    wrapper = mount(BoxBuster, { localVue })
+  it('sorts the list of plates by count_must_sequence', async () => {
+    const sortedExamplePlates = wrapper.vm.sortedPlates(examplePlates)
+    const data = { plates: sortedExamplePlates }
     await wrapper.setData(data)
     const rows = wrapper.find('table').findAll('tr')
-    expect(rows.at(1).text()).toContain('AP-rna-1-8-posi')
-    expect(rows.at(2).text()).toContain('AP-rna-4-5-posi')
-    expect(rows.at(3).text()).toContain('AP-rna-2-2-posi')
-    expect(rows.at(4).text()).toContain('AP-rna-5-0-posi')
-    expect(rows.at(5).text()).toContain('AP-rna-3-0-nmap')
+    expect(rows.at(1).text()).toMatch(/AP-rna-2-0-10-8/)
+    expect(rows.at(2).text()).toMatch(/AP-rna-1-1-5-0/)
+    expect(rows.at(3).text()).toMatch(/AP-rna-0-2-8-6/)
+    expect(rows.at(4).text()).toMatch(/AP-rna-0-1-2-1/)
+    expect(rows.at(5).text()).toMatch(/AP-rna-0-1-0-1/)
+    expect(rows.at(6).text()).toMatch(/AP-rna-no_map/)
   })
 
   it('renders a summary of plates', async () => {
-    const data = { plates: examplePlates }
-    const expected = squish(`Box Summary: Total of ${expectedPlateTotal} plates in box;
-    ${expectedMapTotal} plates with plates maps,
-    ${expectedMaplessTotal} without.
-    Total ${expectedPositiveTotal} positives.`)
-    wrapper = mount(BoxBuster, { localVue })
-    await wrapper.setData(data)
-    const summary = squish(wrapper.find('caption').text())
-    expect(summary).toBe(expected)
+    await wrapper.setData({ plates: examplePlates })
+
+    const caption = squish(wrapper.find('caption').text())
+
+    expect(caption).toContain('Total of 6 plates in box')
+    expect(caption).toContain('5 plates with plate maps')
+    expect(caption).toContain('1 without plate map files')
+    expect(caption).toContain('Total of 25 fit to pick samples')
+    expect(caption).toContain('2 plates with samples that must be sequenced')
+    expect(caption).toContain('4 plates with samples that should preferentially be sequenced')
   })
 
   it('makes it easy to see when plates have a plate map', async () => {
-    const data = { plates: [plateA] }
-    wrapper = mount(BoxBuster, { localVue })
-    await wrapper.setData(data)
-    const row = wrapper
-      .find('table')
-      .findAll('tr')
-      .at(1)
-    expect(row.text()).toBe('AP-rna-1-8-posiYes8')
+    await wrapper.setData({ plates: [plateA] })
+    const row = wrapper.find('table').findAll('tr').at(1)
+    expect(row.text()).toContain(plateA.plate_barcode)
```

Comment:
If this test is checking plate map, I think it should be checking whether it has the string 'Yes' or 'No'

Reply:
Good spot, have added extra test.